app/src/main/res/layout/fragment_complaint_status.xml

```diff
@@ -0,0 +1,305 @@
+            android:id="@+id/includedToolbar"
+            layout="@layout/toolbar_layout"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <ScrollView
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/includedToolbar">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/complaintStatus_ConstraintLayout"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:paddingBottom="20dp">
+
+                <View
+                    android:id="@+id/view1"
+                    android:layout_width="match_parent"
+                    android:layout_height="4dp"
+                    android:background="@color/black_with_opacity_20"
+                    app:layout_constraintTop_toTopOf="parent"
+                    tools:layout_editor_absoluteX="0dp" />
+
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:id="@+id/constraintLayout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:paddingHorizontal="16dp"
+                    android:paddingTop="10dp"
+                    android:paddingBottom="20dp"
+                    app:layout_constraintTop_toBottomOf="@+id/view1"
+                    tools:layout_editor_absoluteX="0dp">
+
+                    <TextView
+                        android:id="@+id/categoryTV"
+                        style="@style/OpenSansSemiBold20sp"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:layout_marginHorizontal="5dp"
+                        android:layout_marginTop="10dp"
+                        tools:text="@string/category"
+                        android:textColor="@color/black"
+                        app:layout_constraintEnd_toStartOf="@+id/cardTickedID"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent" />
+
+                    <TextView
+                        android:id="@+id/localityTV"
+                        style="@style/OpenSansRegular16sp"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="5dp"
+                        tools:text="@string/locality"
+                        android:textColor="@color/grey_4F4F4F"
+                        app:layout_constraintEnd_toEndOf="@+id/categoryTV"
+                        app:layout_constraintHorizontal_bias="0.0"
```

Comment:
you can remove layout_constraintHorizontal_bias

Reply:
Resolved.